antiope-sns/src/Antiope/SNS.hs

```diff
@@ -0,0 +1,22 @@
+module Antiope.SNS
+( MonadAWS
+, publishMsg
+, subscribeMsg
+, module Network.AWS.SNS
+) where
+
+import Control.Lens
+import Data.Text       (Text)
+import Network.AWS     (MonadAWS, send)
+import Network.AWS.SNS
+
+publishMsg :: MonadAWS m => Text -> Text -> m ()
+publishMsg topicArn message = do
+  _ <- send $ publish message & pTopicARN ?~ topicArn
```

Comment:
If we don't consume the response, then I think we could hit the unresolved `ResourceT` issue.

To test, send a few thousand messages and inspect the process with `lsof` and look for thousands of left-over connections.

Reply:
Good point.

Reply:
How did we fix that issue before?

Reply:
We consumed the response :)

i.e. `ackMessage -> m` in SQS is now defined as `ackMessages [m]`, which reads the responses through the batch api.

Reply:
Thanks.